management/appointments/appointmentsController.js

```diff
@@ -21,11 +22,12 @@ define('appointmentsController', [], function() {
 					if (result.appointments.length > 0) {
 						$scope.appointments = result.appointments.map((appointment) => {
 							appointment.name = appointment.firstName + " " + appointment.lastName;
+
 							appointment.completed = appointment.completed == true; // Do this since the SQL returns 0/1, and we want it to be false/true
-							
-							appointment.cancelled = appointment.notCompletedDescription === "Cancelled Appointment";
+							appointment.cancelled = appointment.cancelled == true; // Do this since the SQL returns 0/1, and we want it to be false/true
 							appointment.notStarted = !appointment.cancelled && appointment.timeIn == null;
-							appointment.incomplete = !appointment.cancelled && appointment.notCompletedDescription != null;
+							// TODO: Not sure this logic for incomplete is good
+							appointment.incomplete = !appointment.cancelled && !appointment.completed;
```

Comment:
@CollinsSpencer Can you independently confirm this logic? I don't think it's actually sufficient, but I'm unsure of how to deem an appointment incomplete now that we don't have the notCompletedDescription